tests/e2e/run-e2e-tests.sh

```diff
@@ -129,6 +129,9 @@ if [[ ! -f "/i-am-rox-ci-image" ]]; then
     kubeconfig="${KUBECONFIG:-${HOME}/.kube/config}"
     mkdir -p "$QA_TEST_DEBUG_LOGS"
     info "Running in a container..."
+    # .gitconfig needs to mounted from a location that is shared with the
+    # linux VM running on macOS. Hence we cannot use /tmp/gitconfig here.
+    printf '[safe]\n\tdirectory = "%s"' "${ROOT}" > "$ROOT/.gitconfig"
```

Comment:
Perhaps also include a short comment in the created file pointing to the place where it's being created.